### DISCUSSION TITLE
How can I get an error in my methods ?

### DISCUSSION THREAD

**Discussion by SamoylenkoAA** [react-hook-form/react-hook-form#11560] [Q&A]
I get an empty object with errors

```
const footerActions = useMemo(() => (
    <Button
      type='submit'
      title='next'
      onClick={handleSubmit(onSubmitCallback, () => handleError(method.formState.errors))}
    >
      Next
    </Button>
  ), [handleSubmit, onSubmitCallback, method])

```

**Comment by Moshyfawn** [comment]
`formState` is a `Proxy` object for performance reasons. Please refer to [the` formState` Rules section](https://react-hook-form.com/docs/useform/formstate#rules) of the documentation to learn more about its usage.

**Reply by SamoylenkoAA** [reply]
So it is impossible to get an object with errors except for tracking in useEffect ?

**Reply by Moshyfawn** [reply]
To subscribe your component to the `errors` state changes, you should use destructuring to get the object.

```js
const { errors } = methods.formState
```

**Reply by SamoylenkoAA** [reply]
```
const submitingHandler = useCallback(() => {
    const { errors } = formMethods.formState

    console.log(errors) //empty

   ...
  }, [formMethods])
```
